src/igvfd/types/file_set.py

```diff
@@ -666,16 +666,20 @@ class ConstructLibrarySet(FileSet):
         Path('lab', include=['@id', 'title']),
         Path('submitted_by', include=['@id', 'title']),
         Path('files', include=['@id', 'accession', 'aliases', 'content_type', 'file_format', 'upload_status']),
+        Path('integrated_content_files', include=['@id', 'accession',
+             'aliases', 'content_type', 'file_format', 'upload_status']),
         Path('control_for', include=['@id', 'accession', 'aliases']),
         Path('associated_phenotypes', include=['@id', 'term_id', 'term_name']),
         Path('small_scale_gene_list', include=['@id', 'geneid', 'symbol', 'name', 'synonyms']),
         Path('applied_to_samples', include=['@id', '@type', 'accession',
```

Comment:
Does this array need to include "donors" if the facet path is "applied_to_samples.donors.taxa"?

Reply:
You are correct for both, thanks for catching this. Fixed.